devito/nodes.py

```diff
@@ -297,6 +297,10 @@ def ccode(self):
         """
         loop_body = [s.ccode for s in self.nodes]
 
+        # Disregard offsets for buffered dimensions
+        if self.dim.is_Buffered:
+            self.offsets = [0, 0]
```

Comment:
And is it always the case that ``offsets == [0,0]`` with ``BuffereredDimension``s ?